### DISCUSSION TITLE
To perform Quality control

### DISCUSSION THREAD

**Discussion by Rajeshwarira** [GuyTeichman/RNAlysis#20] [Q&A]
Hello Guy,
I'm a beginner in bioinformatics.
Can you please tell me how can I run a FastQC or other similar tools to run a quality control 
using RNAlysis?

(I use RNAlysis version 3.7.0 in Windows 11).

**Comment by Rajeshwarira** [comment]
I will be glad if you can help me with this as faster as you can sir. If the quality control option is not in RNAlysis, please do guide me what are other ways I can use sir.

**Comment by GuyTeichman** [comment]
Hi Rajeshwarira,
Currently, FastQC is not integrated into RNAlysis. However, FastQC has a friendly graphical interface, which you can download here:
https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
And follow the online tutorial available here:
https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

Feel free to message again if you run into any issues.

Best,
Guy.
